repos.yaml

```diff
@@ -949,6 +949,11 @@ repositories:
         name: Apache Camel Team
       - email: nferraro@apache.org
         name: Nicola Ferraro
+  - name: pgadmin4
+    url: https://github.com/rowanruseler/pgAdmin4
```

Comment:
Thanks for the PR. The URL needs to be to a Helm repository containing the chart. We have documentation on doing that at https://github.com/helm/hub/blob/master/docs/github.md and there is a GitHub action to help with it. You can read about that at https://github.com/marketplace/actions/helm-chart-releaser.